shared/cryptography/codeql/cryptography/Model.qll

```diff
@@ -2365,7 +2416,10 @@ module CryptographyBase<LocationSig Location, InputSig<Location> Input> {
 
     override string getAlgorithmName() { this.hashTypeToNameMapping(this.getHashFamily(), result) }
 
-    int getDigestLength() { result = instance.asAlg().getDigestLength() }
+    int getDigestLength() {
```

Comment:
Same issues as for `getKeySizeFixed` above: "we should unify getDigestLength and getKeySizeFixed as well as the Length/Size/Fixed/ terminology" and concerns about inconsistent modelling paths.